app/Http/Controllers/PhotoController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Photo;
+use Illuminate\Support\Facades\Input;
+
+class PhotoController extends Controller
+{
+    public function index()
+    {
+        return Photo::getLastRecord();
+    }
+
+    public function all(\Illuminate\Http\Request $request)
+    {
+        return Photo::getAllRows($request);
+    }
+
+    public function savePhoto(\Illuminate\Http\Request $request)
+    {
+        $value = Photo::processImageFromApi($request);
+
+        if ($value == true) {
+            return "ALL OK!";
```

Comment:
Return HTTP response status (200, 500) instead of string

Reply:
edited -> return response()->json("ok", 200);